CONTRIBUTING.md

```diff
@@ -96,6 +96,64 @@ Closes #123
 
 In this case 123 is the corresponding issue number.
 
+### When End-To-End Tests Fail
+
+Every pull request triggers a full run of testing in the CI system.
+The failures reported by the code style checker (aka the "linter") and the unit tests are usually
+clear and easy to fix (and can be avoided by running `npm t` locally before creating a commit).
```

Comment:
Optional:
```suggestion
clear and easy to fix (and can be avoided by running `npm test` locally before creating a commit).
```
This makes it clearer for people-new-to-npm what is going on.

Reply:
true